Swe2d

eparish1 opened the pull request.

fnrizzi reviewed (changes requested):
- please remove the DS store file

- i don't mean to be picky :) but just to enforce good practice, can you please review the code you added to make sure you have "const" correctness where it is appropriate? 

- this is just a detail but it might be useful for later to change "rusanovflux_eigen" to remove the "eigen" from the name since those functions do not have anything specific to eigen. 

- what is the difference between "rusanovflux_eigen" and "rusanovflux_kernel"? 
and same question for jacobian?

eparish1 commented:
Thanks for the constructive comments! I have gone through and modified the code accordingly. Per your comments on the flux schemes, I have changed the names to something more appropriate. The difference in the functions is how the states are accessed. In one, we pass in the local left and right state (e.g., in an array with three entries), while in the other we pass in the full state and then index it in the right locations. I had been using the first one for tpetra_block, where it is straightforward to get a pointer to a given element. For Eigen I was using the second one, where it is more intuitive to work on the global array.
